src/community/teams_responsibilities.rst

```diff
@@ -0,0 +1,39 @@
+Teams and responsibilities
+--------------------------
+
+SKA software development is organized with agile development teams.
+
+Development team
+================
+
+
+Scrum Master
+============
+The Scrum Master of each team is the responsible for the process the team
+follows. A generic description of this role can be found `on the SAFe website <https://www.scaledagileframework.com/scrum-master/>`_.
+The SKA Scrum Masters are responsible for:
+
+  *  Meet the team, make sure they know each other and find a nice way to present interests, skills and get to know each other. Lead the team to find a name they like.
+
+  * Make sure all team components can access SKA confluence and jira.
```

Comment:
team members...not team components
Should probably add video conferencing facilities to this list.